### PR TITLE
[C-364, C-374, C-430, C-435] Antd package update

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
### Description

Upgrade antd to v4 and restyle all of the components that were changed so that they conform to our current styles

There were lots of updates here, but they have been thoroughly QA to ensure that there are no regressions or things that look awful

### Dragons

N/A

### How Has This Been Tested?
Lots of manual testing and QA by the product team

### How will this change be monitored?

N/A

### CODE REVIEW

File: packages/web/src/components/data-entry/DropdownInput.module.css [hunk 1]
```diff
@@ -167,32 +195,40 @@
...
   border-radius: 4px;
   font-size: var(--font-xs);
   font-weight: var(--font-medium);
   color: var(--neutral);
   transition: all ease-in-out 0.04s;
 }
-.select :global(.ant-select-dropdown-menu-item-selected) {
-  background: var(--neutral-light-10);
+
+/* .select :global(.ant-select-item-option-active) {
```
Comment (sliptype) [inline packages/web/src/components/data-entry/DropdownInput.module.css:215]: Can we delete this one?

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/kj-Antd-update

**Review by sliptype — APPROVED** [review]
Nice! lgtm

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/kj-Antd-update

File: packages/web/src/components/tooltip/Tooltip.js [hunk 2]
```diff
@@ -98,6 +108,7 @@ Tooltip.propTypes = {
 }
 
 Tooltip.defaultProps = {
+  color: themeColors['--secondary-transparent'],
```
Comment (raymondjacobson) [inline packages/web/src/components/tooltip/Tooltip.js:111]: nice. maybe we should just move this guy to typescript and properly type/default the props

**Review by raymondjacobson — APPROVED** [review]

**Review by dylanjeffers — APPROVED** [review]
amazing job here

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/kj-Antd-update

**Pull request merged**